src/path.js

```diff
@@ -415,6 +415,21 @@ Path.prototype.quadTo = Path.prototype.quadraticCurveTo = function(x1, y1, x, y)
     });
 };
 
+/**
+ * @function drawImage
+ * @memberof opentype.Path.prototype
+ */
+Path.prototype.drawImage = function(image, x, y, width, height) {
+    this.commands.push({
```

Comment:
I don't like the idea of handling this as a Path command. The commands should be actual commands for drawing outlines. I'd prefer to handle this spearately, like I did for colr/cpal drawing. Also the outlines shouldn't be drawn below the SVG image at all, rather if a glyph has an SVG (or COLR/CPAL) representation, that should entirely replace the base glyph's outline.

I'd also suggest the following: If a glyph has COLR layers as well as an associated SVG, I'd say we prefer the SVG version, as we currently only support COLRv0 with solid colors. Unless `options.drawSVG` is set to `false` of course. That way, by turning it off, you can force the COLRv0 version, or switch that off as well in order to render the outlines.

Reply:
What do you think about making a layer with an SVG image? Something like this
```js
const p = new Path();
if ( options.drawLayers ) {
  const svgImage = this.getSvgImage(font);
  if ( svgImage ) {
      const layer = new Path();
      layer._image = {
          image: svgImage.image,
          x: x + svgImage.leftSideBearing * scale,
          y: y - svgImage.baseline * scale,
          width: svgImage.image.width * scale,
          height: svgImage.image.height * scale,
      };
      p._layers = [layer];
      return p;
  }
  const layers = this.getLayers(font);
  // ...
}
```
Or do you prefer to keep images in a separate array?

Reply:
I think we could do it that way! We should just make sure that there are separate options for colr/cpal emojis and svg emojis to toggle and that svg is preferred if both exist for a glyph and svg rendering is not disabled

Reply:
Done c1bd51e785635739c405fe8de2bc3d03a8460c78